spec/order_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'order'
+
+describe Order do
+    it "raises error when item is not displayed in menu" do
+        expect {subject.add_item('Bread', 2).to raise_error('This item is not on the menu')}
```

Comment:
The `expect { ... }` curly braces are in the wrong place here. Take a look at the RSpec docs and see if you can fix it: https://relishapp.com/rspec/rspec-expectations/docs/built-in-matchers/raise-error-matcher

Make sure you can make the test fail by changing the expectation.

Reply:
Hi thanks for this I've moved the braces to after the add item arguments where it should be.